### ISSUE TITLE
support for ios < 14

### ISSUE THREAD

**Issue by seraph3b** [prebid/prebid-mobile-ios#330]
with 1.8.1 app doesn't run on device running ios < 13 due to AppTrackingTransparency framework missing:

error on debug: dyld: Library not loaded: /System/Library/Frameworks/AppTrackingTransparency.framework/AppTrackingTransparency

**Comment by yoalex5** [comment]
Which version of Xcode do you use ?

**Comment by ppuviarasu** [comment]
@yoalex5 anything less than 12 would break... I am using xcode 11.6

**Comment by seraph3b** [comment]
xcode 12.0.1.
I tryed with both simulator and real device and I got crash at app start only with ios < 14. 
version 1.8 works fine

**Comment by yoalex5** [comment]
@ppuviarasu, I assume that your PR #329 fix this problem

**Comment by seraph3b** [comment]
@yoalex5 I agree with that solution, thanks

**Comment by ppuviarasu** [comment]
Yes it should. We just need to add the conditional check

**Comment by ppuviarasu** [comment]
@seraph3b can you try the PR #329 & let us know if that fixes ur problem?

**Comment by yoalex5** [comment]
I use Xcode 12.0 and Simulator iOS 12 - everything  fine
I am downloading Xcode 11.7 and also will check

**Comment by seraph3b** [comment]
> @seraph3b can you try the PR #329 & let us know if that fixes ur problem?

yes of course, I will modify requestBuilder.swift adding:
#if canImport(AppTrackingTransparency)
import AppTrackingTransparency
#endif

at line 501 I think it's ok this:
  if #available(iOS 14, *) {
            deviceExt["atts"] = ATTrackingManager.trackingAuthorizationStatus.rawValue
        }

**Comment by ppuviarasu** [comment]
```#if canImport(AppTrackingTransparency)
if #available(iOS 14, *) {
deviceExt["atts"] = ATTrackingManager.trackingAuthorizationStatus.rawValue
}
#endif```

**Comment by yoalex5** [comment]
@seraph3b, do you have test results ?

@ppuviarasu, are you able to reproduce this runtime issue ?

**Comment by ppuviarasu** [comment]
Fixed with issue #1.8.2

**Comment by seraph3b** [comment]
@yoalex5 well I just deintegrate and then update pods and all worked fine.
it seems that for some reasons AppTrackingTransparency was added to other flags.
thanks for your support
